orchagent/aclorch.cpp

```diff
@@ -1392,6 +1407,10 @@ bool AclTable::create()
         table_attrs.push_back(attr);
     }
 
+    attr.id = SAI_ACL_TABLE_ATTR_FIELD_OUTER_VLAN_ID;
```

Comment:
So this is expected to be set for all tables except PFC_WD and ACL_TABLE_MIRROR_DSCP?

Reply:
L3, L3V6, MIRROR, and MIRRORV6. DSCP and PFCWD are skipped in this method, and MCLAG and DTEL use their own init methods.